metadata-ingestion/tests/integration/delta_lake/minio/setup_minio.sh

```diff
@@ -1,6 +1,15 @@
+INTEGRATIONS_DIR="./tests/integrations/minio"
 #ref https://hub.docker.com/r/minio/minio/
-
-wget https://dl.min.io/server/minio/release/linux-amd64/minio -P ./tests/integrations/delta_lake/minio/
-chmod +x ./tests/integrations/delta_lake/minio/minio
-nohup ./tests/integrations/delta_lake/minio/minio server ./tests/integrations/delta_lake/minio/data > temp.log 2>&1 &
-echo $! > ./tests/integrations/delta_lake/minio/minio_pid.txt
+if [[ $(uname -m) == 'arm64' && $(uname) == 'Darwin' ]]; then
```

Comment:
Why we don't install with brew if the arch is `Darwin` and `amd64`?

FYI -> here is how we do brew install in the preflight script -> 
https://github.com/datahub-project/datahub/blob/cfa864e4187b6f9adafd7e951294be8e6109d03d/metadata-ingestion/scripts/datahub_preflight.sh#L8C7-L8C7

Reply:
I was just trying to make work with arm macs. I don't really have a preference here